en-devonfw.dict

```diff
@@ -17,7 +17,6 @@ JSR
 Balancer
 cdk
 Subclasses
-IServiceCollection
 util
 JTA
 ifdef
```

Comment:
Delete?

Reply:
yes should be part covered in backticks part